packages/rest/src/samples/sample.ts

```diff
@@ -1,10 +1,9 @@
 import { LogLevel, AutoAcceptCredential } from '@aries-framework/core'
 import { connect } from 'ngrok'
 
-import { setupAgent } from './utils/agent'
-import { TestLogger } from './utils/logger'
-
-import { startServer } from './index'
+import { setupAgent } from '../../tests/utils/agent'
+import { TestLogger } from '../../tests/utils/logger'
+import { startServer } from '../index'
 
```

Comment:
Sorry for the nitpick, but with the samples directory I meant placing it outside the `src` directory. Otherwise it gets published as part of the npm package.

```
packages/
  rest/
    src/
     index.ts
    samples/
      sample.ts
```